src/backend/expungeservice/models/case.py

```diff
@@ -97,5 +105,6 @@ def create(
 
     @staticmethod
     def compute_balance_due_in_cents(balance_due_dollar_amount: str):
-        balance_due_dollar_amount_float = float(balance_due_dollar_amount.replace(",", ""))
-        return int(balance_due_dollar_amount_float * 100)
+        is_negative = len(balance_due_dollar_amount) > 0 and balance_due_dollar_amount[0] == "("
+        balance_due_dollar_amount_float = float(re.sub("[,()]", "", balance_due_dollar_amount))
+        return (-1 if is_negative else 1) * int(balance_due_dollar_amount_float * 100)
```

Comment:
```suggestion
        return int(CaseCreator._balance_to_float(balance_due_dollar_amount) * 100)

    @staticmethod
    def _balance_to_float(balance: str) -> float:
        commas_removed = balance.replace(",","")
        normalized_negative = re.sub("\((?P<balance>.*)\)", "-\g<balance>", commas_removed)
        return float(normalized_negative)
```

Reply:
This feels more "proper" than `balance_due_dollar_amount[0] == "("` but it might be harder to read as no where else in the codebase do we use match groups in regex. Feel free to ignore.

Reply:
Mmmmm. How does one learn this black magic?

Reply:
Though it does feel cleaner and not crazy to figure out what's going on.